tests/imposter/data/questions.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
We probably don't want to expose the underlying API structure to our frontends - a simplified generic "free text question" would be useful.

But until we've made that decision, including the content here make the most sense :+1: